src/test/java/com/google/firebase/auth/FirebaseAuthIT.java

```diff
@@ -320,9 +320,118 @@ public void onSuccess(ListUsersPage result) {
     }
   }
 
+  @Test
+  public void testTenantLifecycle() throws Exception {
+    TenantManager tenantManager = auth.getTenantManager();
+
+    // Create tenant
+    Tenant.CreateRequest createRequest = new Tenant.CreateRequest().setDisplayName("DisplayName");
+    Tenant tenant = tenantManager.createTenantAsync(createRequest).get();
+    String tenantId = tenant.getTenantId();
```

Comment:
Do some basic assertions on the returned Tenant here.

Reply:
Done.